### PR TITLE
Issue 1591 - Add support to send chunk info in JSON format with each …

### PR DESCRIPTION
**Pull request by krraghavan**
## Brief description of the changes [REQUIRED]

This PR allows clients to use the onUploadChunk callback method to customize the payload sent with each chunk (Issue 1591)
## What browsers and operating systems have you tested these changes on? [REQUIRED]

IE11, Safari, Firefox, Chrome
## Are all automated tests passing? [REQUIRED]

Local build succeeded.
## Is this pull request against develop or some other non-master branch? [REQUIRED]

No

…chunk

### CODE REVIEW

**Comment by rnicholus** [comment]
It looks like you ran into some issues while implementing this. Do you have plans to continue your work?

**Comment by krraghavan** [comment]
No I am using my fork in our project. Works great. Have to fix unit tests. Will do that in the next week or so and get back to you. My changes broke the unit tests and I'll also look to add more for my changes
Thx for following up
Raghavan. 

Please pardon my brevity and any typos as this mail is sent from my mobile device

**Comment by krraghavan** [comment]
Hi Ray - is there anything more you need from me before you merge this in?

**Comment by rnicholus** [comment]
Sorry for the delay. I still need to spend some time looking into your proposed changes and running some manual regression tests cross-browser. Also, unit tests and documentation will need to be written. There are a number of other issues and tasks in the queue ahead of this one at the moment, and I'll be on vacation for a couple weeks. I'll try to get to this as soon as reasonably possible, but I don't have a date for you at this time.

File: client/js/uploader.basic.js [hunk 1]
```diff
@@ -24,7 +24,8 @@
                 params: {},
                 paramsInBody: true,
                 totalFileSizeName: "qqtotalfilesize",
-                uuidName: "qquuid"
+                uuidName: "qquuid",
+                chunkInfoParamName: "qqchunkinfo"
```
Comment (rnicholus) [inline client/js/uploader.basic.js]: I'd like to keep these params alphabetically ordered.

**Review by rnicholus — CHANGES_REQUESTED** [review]
A couple things:

1. What exactly is a "chunk ID"?
2. Can you add unit tests & documentation?

**Comment by rnicholus** [comment]
closing as abandoned

**Pull request closed without merging**